packages/ingest/src/granule.ts

```diff
@@ -1,9 +1,11 @@
-import * as errors from '@cumulus/errors';
+import * as crypto from 'crypto';
```

Comment:
Why export the method from `ingest` as opposed to something like api/lib/granules? Not saying it's wrong, just thinking through the best spot for this method. It's definitely something that will be used in ingest but not exclusively.

Reply:
The consideration here, really, is that this isn't a thing the API should be doing - it's a function of task components doing an ingest function. 

That said, it's possible it belongs elsewhere. 😅    I feel pretty strongly that the API is the wrong place for it though.

Reply:
Both ParsePDR and DiscoverGranules import `@cumulus/ingest`. That's a strong enough argument for me to just leave it here. Thanks for thinking about it.